src/main/java/com/aws/greengrass/shadowmanager/sync/strategy/PeriodicSyncStrategy.java

```diff
@@ -28,25 +77,30 @@ public PeriodicSyncStrategy(Retryer retryer) {
      */
     @Override
     public void start(SyncContext context, int syncParallelism) {
-
+        super.startSync(context, syncParallelism);
     }
 
     @Override
     void doStart(SyncContext context, int syncParallelism) {
-
+        logger.atInfo(SYNC_EVENT_TYPE).kv("interval", interval).log("Start periodic syncing");
+        this.scheduledFuture = syncExecutorService
+                .scheduleAtFixedRate(this::syncLoop, 0, interval, TimeUnit.SECONDS);
     }
 
     @Override
     void doStop() {
-
+        logger.atDebug(SYNC_EVENT_TYPE).log("Cancel {} periodic sync thread(s)", syncThreads.size());
```

Comment:
nit: log the start and stop at the same log level. log is at Info, stop is at Debug. Maybe add a separate info log that you that says "Stop periodic syncing")?